libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFSmartStoreTests.m

```diff
@@ -40,27 +38,19 @@
 #import <SalesforceSecurity/SFPasscodeManager+Internal.h>
 #import <SalesforceSecurity/SFPasscodeProviderManager.h>
 #import "SFSecurityLockout+Internal.h"
-#import "SFUserAccountManager.h"
 #import <SalesforceSecurity/SFKeyStoreManager.h>
 #import <SalesforceSecurity/SFEncryptionKey.h>
 #import <SalesforceCommonUtils/NSString+SFAdditions.h>
 #import <SalesforceCommonUtils/NSData+SFAdditions.h>
 
-NSString * const kTestSmartStoreName   = @"testSmartStore";
-NSString * const kTestSoupName   = @"testSoup";
+#define kTestSmartStoreName  @"testSmartStore"
```

Comment:
That way we don't need to have constant names that are unique across the entire compilation unit.